roboteam_ai/test/ConditionTests/IsInDefenseAreaTest.cpp

```diff
@@ -38,19 +38,19 @@ TEST(DetectsInOurDefenseArea,IsInDefenseAreaTest){
     rtt::ai::World::set_world(worldMsg);
 
     // Should succeed since robot is in our defence area
-    ASSERT_EQ(node.update(), bt::Node::Status::Success);
+    EXPECT_EQ(node.update(), bt::Node::Status::Success);
```

Comment:
Why are this suddenly expects instead of asserts?

Reply:
We generally prefer to use expects instead of asserts so that we can get multiple errors from running 1 test (if more things break). I'd figure I'd change it when I was going over the tests and had to refactor them so I could see if more things were breaking.